ios/RNGalleryManager.m

```diff
@@ -130,19 +138,10 @@ @implementation RNGalleryManager
   NSArray<PHAsset*>  *results = [fetchResults objectsAtIndexes:indexSet];
   
   for (PHAsset* asset in results) {
-    NSArray *resources = [PHAssetResource assetResourcesForAsset:asset ];
-    if ([resources count] < 1) continue;
-    NSString *orgFilename = ((PHAssetResource*)resources[0]).originalFilename;
-    NSString *uit = ((PHAssetResource*)resources[0]).uniformTypeIdentifier;
-    NSString *mimeType = (NSString *)CFBridgingRelease(UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassMIMEType));
-    CFStringRef extension = UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassFilenameExtension);
-    
     [assets addObject:@{
                         @"type": [self getMediaType:([asset mediaType])],
                         @"width": @([asset pixelWidth]),
                         @"height": @([asset pixelHeight]),
-                        @"filename": orgFilename ?: @"",
-                        @"mimeType": mimeType ?: @"",
                         @"id": [asset localIdentifier],
                         @"creationDate": [asset creationDate],
                         @"uri": [self buildAssetUri:[asset localIdentifier] extension:extension],
```

Comment:
where is the extension coming from?

Reply:
sorry forgot about the uri. 

if you don't need extension i will make these additional changes

In RNGalleryManager.m 145
@"uri": [NSString stringWithFormat:@"ph://%@", [asset localIdentifier]],

in RctImageLoader.m
```NSString *assetID = [imageURL.absoluteString substringFromIndex:@"ph://".length];
results = [PHAsset fetchAssetsWithLocalIdentifiers:@[assetID] options:nil];```

This is what default RN loader is using.

if you are happy i will make these changes